R/antigen.garnish_predict.R

```diff
@@ -264,12 +263,12 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
 
     blastdt[, iedb_score := SW %>% modeleR, by = "nmer_uuid"]
 
-    blastdt[, highest := max(iedb_score), by = "nmer_uuid"]
+    blastdt[, highest := max(SW), by = "nmer_uuid"]
```

Comment:
max will fail without na.rm

Reply:
Agree, toyed with adding na.rm, but there shouldn't be NA at this point for any reason, so want fail there because I want to know if it starts tossing errors and is a new edge case.

Reply:
great